saleor/csv/utils/__init__.py

```diff
@@ -54,5 +53,6 @@ class ProductExportFields:
         "channel_pk": "variants__channel_listing__channel__pk",
         "slug": "variants__channel_listing__channel__slug",
         "price_amount": "variants__channel_listing__price_amount",
-        "currency": "variants__channel_listing__currency",
+        "variant_currency_code": "channel_listing__channel__currency_code",
```

Comment:
It shouldn't be `"variant_currency_code": "variants__channel_listing__currency"` as `ProductVariantChannelListing` has it own `currency` which is connected with `price_amount`?